python/nav/models/sql/changes/sc.05.13.0001.sql

```diff
@@ -0,0 +1,4 @@
+-- Delete widget_columns key from account preferences
+UPDATE account
+SET preferences = delete(preferences, 'widget_columns')
+WHERE preferences ? 'widget_columns';
```

Comment:
Since our migration system refers to release version numbers (more by convention than anything else), and we've missed the deadline for 5.13, and this should be named 5.14-something.  Beware of conflicts with #3268